azavea/django-ecsmanage
# Migrate from Travis to GitHub Actions

## Overview

Ports Travis CI config to a GitHub Actions workflow.

Closes #20 

### Notes

The `PYPI_AZAVEA_USERNAME` and `PYPI_AZAVEA_PASSWORD` secrets have been made available to this repository.

## Testing Instructions

 * Verify that the [GitHub Actions CI workflow](https://github.com/azavea/django-ecsmanage/actions/runs/173191359) passes.
* Verify that [this Release workflow](https://github.com/azavea/django-ecsmanage/pull/22/checks?check_run_id=883504715) ran. The workflow was expected to fail at the "Build and publish package" step since the PyPI credentials had not been shared with this repository yet.
